An error occured. This is either a bug in UltiSnips or a bug in a
snippet definition. If you think this is a bug, please report it to
https://github.com/SirVer/ultisnips/issues/new.

Following is the full stack trace:
Traceback (most recent call last):
  File "/home/yabuki/.vim/pythonx/UltiSnips/snippet_manager.py", line 61, in wrapper
    return func(self, *args, **kwds)
  File "/home/yabuki/.vim/pythonx/UltiSnips/snippet_manager.py", line 163, in expand
    if not self._try_expand():
  File "/home/yabuki/.vim/pythonx/UltiSnips/snippet_manager.py", line 725, in _try_expand
    self._do_snippet(snippet, before)
  File "/home/yabuki/.vim/pythonx/UltiSnips/snippet_manager.py", line 679, in _do_snippet
    si = snippet.launch(text_before, self._visual_content,
  File "/home/yabuki/.vim/pythonx/UltiSnips/snippet/definition/_base.py", line 421, in launch
    snippet_instance.update_textobjects()
  File "/home/yabuki/.vim/pythonx/UltiSnips/text_objects/_snippet_instance.py", line 79, in update_textobjects
    if obj._update(done):
  File "/home/yabuki/.vim/pythonx/UltiSnips/text_objects/_python_code.py", line 271, in _update
    exec(code, self._locals)  # pylint:disable=exec-used
  File "<string>", line 2, in <module>
  File "<string>", line 51, in make_box
  File "<string>", line 47, in _get_comment_format
  File "<string>", line 15, in _parse_comments
ValueError: not enough values to unpack (expected 2, got 1)

Executed snippet code:
  1
  2   box = make_box(len(t[1]))
  3   snip.rv = box[0] + '\n' + box[1]
  4
